apps/eas-expo-go/eas.json

```diff
@@ -21,7 +21,7 @@
       },
       "ios": {
         "cache": {
-          "key": "sdk47",
+          "key": "sdk47-0.70.2",
```

Comment:
I guess it's something we will need to update every time we update RN, right? Maybe we should do that in the script?

Reply:
it's not exactly for each update. the problem used to break from hermes-engine. if the react-native doesn't update the bundled hermes-engine, that would be safe. i'm still visiting the possibilities of updating the key automatically, will follow up on it in a separated pr.